rpms/build.sh

```diff
@@ -66,17 +68,38 @@ cp -R wazuh-* ${build_dir}/${package_name}
 # Including spec file
 if [ "${use_local_specs}" = "no" ]; then
     curl -sL https://github.com/wazuh/wazuh-packages/tarball/${wazuh_packages_branch} | tar zx
-    specs_path="wazuh*/rpms/SPECS"
+    specs_path=$(find . -type d -name "SPECS" -path "*rpms*")
 else
     specs_path="/specs"
 fi
 
+if [[ "${future}" == "yes" ]]; then    
+    # PREPARE FUTURE SPECCS
+    base_version=$wazuh_version
+    MAJOR=$(echo $base_version | cut -dv -f2 | cut -d. -f1)
+    MINOR=$(echo $base_version | cut -d. -f2)
+    wazuh_version="${MAJOR}.30.0"
+    file_name="wazuh-${build_target}-${wazuh_version}-${package_release}"
+    old_name="wazuh-${build_target}-${base_version}-${package_release}"
+    package_name=wazuh-${build_target}-${wazuh_version}
+    old_package_name=wazuh-${build_target}-${base_version}
+    cp -r "${specs_path}/${base_version}" "${specs_path}/${wazuh_version}"
+    cd "${specs_path}/${wazuh_version}"
+    rename "${base_version}" "${wazuh_version}" *${base_version}*
+    cd -
```

Comment:
Can you run this without the `cd` commands? Just to make a cleaner code.

Reply:
Same